app/src/main/res/layout/episodes_apply_action_fragment.xml

```diff
@@ -1,115 +1,53 @@
-            android:background="@android:color/transparent"
-            android:drawableTop="?attr/av_download"
-            android:text="@string/download_label"
-            android:textSize="10sp" />
-
-        <View
-            android:id="@+id/divider4"
-            android:layout_width="1dp"
-            android:layout_height="match_parent"
-            android:layout_margin="4dp"
-            android:background="?android:attr/listDivider"
-            tools:background="@android:color/holo_red_dark" />
-
-        <Button
-            android:id="@+id/btnDelete"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_weight="1"
-            android:background="@android:color/transparent"
-            android:drawableTop="?attr/content_discard"
-            android:text="@string/remove_episode_lable"
-            android:textSize="10sp" />
-
-    </LinearLayout>
-
-    <View
-        android:id="@+id/divider"
-        android:layout_width="match_parent"
-        android:layout_height="1dp"
-        android:layout_above="@id/bottomBar"
-        android:background="?android:attr/listDivider"
-        android:paddingBottom="4dp"
-        tools:background="@android:color/holo_red_dark" />
-
     <ListView
         android:id="@android:id/list"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_above="@id/divider"/>
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="0dp" />
+
+    <com.leinardi.android.speeddial.SpeedDialOverlayLayout
+        android:id="@+id/fabSDOverlay"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+    <!-- The FAB SpeedDial
+         1. MUST be placed at the bottom of the layout xml to ensure it is at the front,
+            clickable on Pre-Lollipop devices (that do not support elevation).
+            See: https://stackoverflow.com/a/2614402
+         2. ScrollView is needed to ensure the vertical list of speed dials are
+            accessible when screen height is small, eg., landscape mode on most phones.
+      -->
+    <ScrollView
+        android:id="@+id/fabSDScrollCtr"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:elevation="@dimen/sd_close_elevation"
+        tools:ignore="UnusedAttribute">
+        <!-- android:elevation:
+             1. Needs to match the speed dial's minimal elevation,
+                or the speed dial can't be clicked at all
+             2. It is honored for API >= 21, but it does not hurt to be added here.
+                Use tools:ignore to suppress warnings.
+                Otherwise, we need a custom layout for API >= 21 that is more difficult to maintain.
+          -->
```

Comment:
I don't think this comment is needed

Reply:
Without the comment, another developer may wonder what `tools:ignore="UnusedAttribute"` try to suppress.

Reply:
I usually just keep the warning (without suppressing). Those warnings are everywhere in open-source projects. Reading the comment is more 'mental work' than just ignoring a yellow underline.

Reply:
The comment is removed, but I suppose it is a matter of personal style. For me, those warnings present their own "mental work", as I would then need to think about and decide if the specific warning is safe to ignore.